src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -10,11 +10,13 @@
     <AssemblyName>DSOffice</AssemblyName>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DSOffice.xml</DocumentationFile>
   </PropertyGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
```

Comment:
I understand why we would avoid Excel on linux but why are we limiting it to netframework only ?
Does it not work on dotnet6+ ?

Reply:
Here is something I tried before
https://github.com/DynamoDS/Dynamo/pull/12716/files#diff-a9d51e372ddc4731d6c88a4bfbb2dcd01b5dfa880e0d34d34078f09fd4aaf9a7R30

I think I also added a note about this on the wiki

Reply:
https://www.nuget.org/packages/Microsoft.Office.Interop.Excel#supportedframeworks-body-tab

Reply:
I think I will leave this for another time and file a followup if thats ok.